client/src/templates/Challenges/video/Show.js

```diff
@@ -217,6 +216,21 @@ export class Project extends Component {
                   ))}
                 </div>
                 <Spacer />
+                <div
+                  style={{
+                    textAlign: 'center'
+                  }}
+                >
+                  {this.state.showWrong ? (
+                    <span>
+                      Ugh - Oh! Unfortunately that answer is not correct. Try
+                      again?
```

Comment:
```suggestion
                      Sorry, that's not the right answer.  Give it another try?
```
I'm not sure what the best wording is, but I think this is a little better.